x/oracle/keeper/oracle.go

```diff
@@ -0,0 +1,74 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+)
+
+func (k Keeper) RegisterOracle(ctx sdk.Context, msg *types.MsgRegisterOracle) error {
```

Comment:
I'll add it to check if it's already added as Oracle.

Reply:
fix: https://github.com/medibloc/panacea-core/pull/519/commits/b3a30fb4ecb2cf072fd7b3231bfa0423c9152c4e